doc/2/plugins/guides/events/kuzzle-start/index.md

```diff
@@ -6,6 +6,24 @@ title: kuzzle:start
 
 # kuzzle:start
 
+<DeprecatedBadge version="change-me" />
+
+Triggered when Kuzzle has finished its startup sequence, and is about to accept network connections.
+
+Kuzzle will wait for pipe listeners to return before accepting user requests.
+
+
+## kuzzle:start:before
+
+<SinceBadge version="change-me" />
+
 Triggered when Kuzzle has finished its startup sequence, and is about to accept network connections.
 
 Kuzzle will wait for pipe listeners to return before accepting user requests.
+
+## kuzzle:start:after
+
+<SinceBadge version="change-me" />
```

Comment:
Now we are sure it's gonna be a new feature but in general you don't know exactly the next Kuzzle version when you make your PR because it could be an enhancement but then a PR with a new feature is merged and the next release will be a minor and not a patch.
We should change those tag in the release tool IMHO

Reply:
I like that idea, and that shouldn't be too hard to code either.

Reply:
I will modify the release-tool with the next release :+1: